internal/db/namespaced.go

```diff
@@ -29,6 +30,15 @@ func NewNamespacedKV(db *leveldb.DB, prefix string) *NamespacedKV {
 	}
 }
 
+// Reset removes all entries in this namespace.
```

Comment:
You should rebase your branch ontop of master, rather than oulling master into your branch.